packages/@react-spectrum/datepicker/stories/DateField.stories.tsx

```diff
@@ -117,6 +117,10 @@ storiesOf('Date and Time/DateField', module)
   .add(
     'placeholderValue: 1980/1/1, zoned',
     () => render({placeholderValue: toZoned(new CalendarDate(1980, 1, 1), 'America/Los_Angeles')})
+  )
+  .add(
+    'showFormatHelpText',
+    () => render({showFormatHelpText: true})
```

Comment:
Looks like the help text here doesn't have an id? Doesn't look like the field has an `aria-describedby` attached either. I assume it should?
![image](https://user-images.githubusercontent.com/9661127/139757569-da325c24-3785-4688-8e62-3c35ffe4caf6.png)

Reply:
This is intentional. See comment here: https://github.com/adobe/react-spectrum/pull/2510/files#diff-3017d859853840f0d9d9723007e3cebeb71da9564d5ff4b6f32e7d04a7a690b0R47-R52